PRS5001-6000/PRS5782JuelJen.xml

```diff
@@ -36,22 +36,77 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </profileDesc>
       <revisionDesc>
          <change who="PL" when="2016-04-28">batch updated according to requirements and issues.</change>
-         <change who="PL" when="2016-03-21"> Created file from
-                                                  google spreadsheet </change>
-         <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="PL" when="2016-03-21"> Created file from google spreadsheet </change>
+         <change who="ES" when="2016-02-09">Created: person</change>
+         <change who="JG" when="2020-04-04">Added information about person and manuscript ownership</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
             <person sex="1">
-               <persName>
-                   Bent
-                  Juel Jensen
-               </persName>
-               <occupation> sch. </occupation>
+               <persName xml:lang="english" xml:id="  n1">Bent Juel-Jensen</persName>
+               <persName type="alt">Bent Einer Juel-Jensen</persName>
+               <nationality type="Denmark"/>
+               <occupation type="medical" subtype="academic"/>
+               <birth>1922</birth>
```

Comment:
Completely fine, but it's always good to mark up dates more precisely with the corresponding attributes, to facilitate later filtering and searching 
`<birth when="1922"/>` or `<birth when="1922">1922</birth> etc.`